src/types/app-manager.ts

```diff
@@ -590,7 +590,7 @@ function getLocalsReference(localsReference: LocalsReference): algosdk.Transacti
 /**
  * Returns an `algosdk.TransactionResourceReference` given a `AccessReference`.
  */
-export function getAccessReference(accessReference: AccessReference): algosdk.TransactionResourceReference {
+export function getTransactionResourceReference(accessReference: ResourceReference): algosdk.TransactionResourceReference {
```

Comment:
```suggestion
export function getResourceReference(accessReference: ResourceReference): algosdk.TransactionResourceReference {
```

Thinking ahead a little. The decoupling name will probably be `getResourceReference` so may as well name as such?